packages/application-shell/src/components/application-shell/redirect-to-login.tsx

```diff
@@ -31,7 +31,7 @@ const generateAndCacheNonceWithState = (state: AuthorizeSessionState) => {
 const RedirectToLogin = () => {
   const location = useLocation();
 
-  if (window.app.__DEVELOPMENT__) {
+  if (window.app.__DEVELOPMENT__?.oidc?.authorizeUrl) {
```

Comment:
The shape of the `__DEVELOPMENT__` field changed a bit (see constants file). Now the OIDC config is scoped under `oidc` field.